src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -111,6 +112,7 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
             throw new MissingDataError(MissingDataErrorType.MissingConsignments);
         }
 
+        const amountTransformer = new AmountTransformer(order.currency.decimalPlaces);
         const consignment = consignments[0];
 
         if (!consignment || !consignment.selectedShippingOption) {
```

Comment:
Will `ShippingOption` be required for Digital Products too?

Reply:
I change the validation, now for digital products only orders an empty string will be sent.